js/cljdoc.tsx

```diff
@@ -15,14 +15,12 @@ import {
   toggleArticlesTip,
```

Comment:
Had to rename this file and `cljdoc.ts` to get the esbuild file name outputs to line up.